src/eos-updater-apply.c

```diff
@@ -73,6 +74,431 @@ get_test_osname (void)
   return g_getenv ("EOS_UPDATER_TEST_UPDATER_OSTREE_OSNAME");
 }
 
+typedef struct
+{
+  char *repo;
+  char *app_identifier;
+  FlatpakRefKind ref_kind;
+} FlatpakToInstall;
```

Comment:
Could we use `FlatpakRemoteRef` instead of inventing a new type?

Reply:
We *could*. I mean, this type basically only exists as a container for parsed out information from the auto-install file.